code/modules/reagents/chemistry/recipes.dm

```diff
@@ -58,6 +58,10 @@
 	///Tagging vars
 	///A bitflag var for tagging reagents for the reagent loopup functon
 	var/reaction_tags = NONE
+	var/centrifuge_recipe = FALSE
+	var/pressure_required = 0
+	var/radioactivity_required = 0
+	var/bluespace_recipe = FALSE
```

Comment:
move these up next to the other requirement variables.

Reply:
I think I did this and forgot to say so